src/parser/jobs/mnk/modules/TwinSnakes.tsx

```diff
@@ -0,0 +1,226 @@
+import {Plural, Trans} from '@lingui/react'
+import React from 'react'
+
+import {ActionLink, StatusLink} from 'components/ui/DbLink'
+import {getDataBy} from 'data'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+
+import {BuffEvent, CastEvent} from 'fflogs'
+import Module, {dependency} from 'parser/core/Module'
+import Checklist, {Requirement, Rule} from 'parser/core/modules/Checklist'
+import Combatants from 'parser/core/modules/Combatants'
+import Enemies from 'parser/core/modules/Enemies'
+import Invulnerability from 'parser/core/modules/Invulnerability'
+import Suggestions, {SEVERITY, Suggestion, TieredSuggestion} from 'parser/core/modules/Suggestions'
+
+import DISPLAY_ORDER from './DISPLAY_ORDER'
+
+const TWIN_SNAKES_CYCLE_BUFFER = 3000
+const TWIN_SNAKES_CYCLE_LENGTH = 6
+
+class TwinState {
+	casts: CastEvent[]
+	start: number
+	end?: number
+
+	constructor(timestamp: number) {
+		this.start = timestamp
+		this.casts = []
+	}
+
+	// Mainly here in case we care about oGCDs being unbuffed later
+	public get gcds(): number {
```

Comment:
```suggestion
	get gcds(): number {
```

Typescript getters are public by default.

Reply:
ack mentioned this but I like being explicit, same for a lot of my returns being typed even when it's easy to infer.